### PR TITLE
Regtest feerate setting

### PR DESCRIPTION
**Pull request by rustyrussell**
We have a dev knob to disable it for testing, but this is what everyone actually wants when using regtest, AFAICT.

During a recent demo I had one regtest peer send out an update_fee with a 0 rate.  To my consternation, I have been unable to reproduce this.  Is bitcoind giving wild fee estimates in some regtest setups?

### CODE REVIEW

File: plugins/bcli.c [hunk 1]
```diff
@@ -994,6 +1015,10 @@ int main(int argc, char *argv[])
 				  " closed more often due to fee fluctuations,"
 				  " large values may result in large fees.",
 				  u32_option, &bitcoind->max_fee_multiplier),
+		    plugin_option("dev-no-fake-fees",
+				  "bool",
+				  "Supress fee faking for regtst",
```
Comment (cdecker) [inline plugins/bcli.c:1020]:
```suggestion
				  "Supress fee faking for regtest",
```

**Review by cdecker — COMMENTED** [review]
Just a typo :-)

ACK https://github.com/ElementsProject/lightning/pull/4405/commits/54d27955c0f410151df860820863536b75db1b19

**Review by cdecker — APPROVED** [review]

File: plugins/bcli.c (hunk 1 above)
Comment (rustyrussell) [reply plugins/bcli.c:1020]: There's also two pp in suppress :(

**Comment by rustyrussell** [comment]
Ack https://github.com/ElementsProject/lightning/commit/6ebbacf935d7d07c69fb1ce6ec5456d28d6f720d

Spelling only

**Pull request merged**